deploy/kustomize/base/operator_group.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kubevirt-hyperconverged-group
+  namespace: kubevirt-hyperconverged
+spec:
+  targetNamespaces:
+    - kubevirt-hyperconverged
```

Comment:
The Subscription and HCO CR also need to consume the namespace, in their `metadata.namespace` fields.
I'll include a patch for them as well.

Reply:
PR is now updated, incl. readme file.

Reply:
I missed it on the first glance but you should be able to specify `namespace: my-custom-namespace` in the kustomization.yaml that matches the metadata.name in the `namespace.yaml` resource specified and all of the subscription, operator group, and CR would be updated to use the correct namespace.

Reply:
I made an update utilizing that feature. I extracted the customized namespace from `namespace.yaml` and use that value to set $TARGET_NAMESPACE (instead of getting from env var) and put that on the base `kustomization.yaml`.